tests/test-maintenance-window.yml

```diff
@@ -0,0 +1,49 @@
+---
+- hosts: localhost
+  connection: local
+  gather_facts: No
+  vars:
+    apikey: "{{ vault_apikey }}"
```

Comment:
Are we happy with leaving reference to vault_apikey in the code for both tests? Other than just clarifying this everything looks good for merging.

Reply:
I've added a comment in the README about it. I think it's ok to leave in. Only people contributing to this repo will want to run these playbooks.